### PR TITLE
Add getRecordings() to update function

### PR DESCRIPTION
**Pull request by Floyer007**
While using the library in HA I just realized I forgot to add getRecordings() to the update function.
Sorry about that.

### CODE REVIEW

**Comment by Floyer007** [comment]
Are you fine with this changes, especially with the test?
I just own a API-level 6 TV, so I made the assumption the structure is the same for API-level 1.

**Comment by elupus** [comment]
V1 does not have this API, so you must make it conditional.

**Comment by Floyer007** [comment]
Just one thing before I start reverting the changes for v1 and make it conditional.

Line 57 in data/v1.py for the test indicates the recordings feature like in v6:
```
        "jsonfeatures": {
            "recordings": ["List", "Schedule", "Manage"],
            "ambilight": ["LoungeLight"],
            "textentry": ["context_based", "initial_string_available"],
            "inputkey": ["key", "unicode"],
            "pointer": ["context_based"],
            "activities": ["browser"],
        },
```
Is this mock-data wrong then?

**Comment by elupus** [comment]
Unclear. At least we don't know how the recording api would look on a v1 series tv. It could be that that section is fully wrong and that TV actually is some higher API version, but the reports we got for it was wrong.

**Comment by Floyer007** [comment]
I see. I limited the feature to API level 6 now, as I can verify it working with my TV.

**Comment by Floyer007** [comment]
Hi, any chance to get this merged?
Am I still missing something?

File: tests/test_v6.py [hunk 1]
```diff
@@ -143,6 +144,9 @@ async def client_mock(param: Param):
             )
         else:
             raise Exception
+        respx.get(f"{param.base}/recordings/list").respond(
+            json=RECORDINGS_LIST
+        )
```
Comment (elupus) [inline tests/test_v6.py:149]: You are not actually testing that this data can be read anywhere.

Comment (Floyer007) [reply tests/test_v6.py:149]: I fixed it by testing selected value combinations

**Comment by Floyer007** [comment]
May I ask for a second review?

**Review by elupus — APPROVED** [review]
Looks good. I think you might need to fix the event function too for this to work when when a user is using eventing to update. But we can look at that later.

**Comment by Floyer007** [comment]
You mean "notifyChange"?
I will look into it.

**Pull request merged**